src/vs/workbench/contrib/interactiveSession/browser/interactiveSessionInputPart.ts

```diff
@@ -123,15 +141,19 @@ export class InteractiveSessionInputPart extends Disposable implements IHistoryN
 		this._inputEditor.focus();
 	}
 
-	async acceptInput(query?: string | IInteractiveSessionReplyFollowup): Promise<void> {
+	async acceptInput(query?: string | IInteractiveSessionReplyFollowup, skipExecution?: boolean): Promise<void> {
```

Comment:
The input editor is already exposed, and you can operate on that directly. I don't think we should add this parameter that totally changes the meaning of the method